### PR TITLE
January post CPU update to test code

### PR DESCRIPTION
**Pull request by sefroberg**
This is the post CPU update to the test code. Included in the update, updated versions of the container tags, updated the readme to include a link to the OpenJDK 21 images and readded the containerfile to support the InterOp testing.

### CODE REVIEW

File: ContainerFile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Create the directory structure for the testing.
+RUN mkdir -p /tmp/rhscl_openshift_dir/openjdk
+WORKDIR /tmp/rhscl_openshift_dir/openjdk
+COPY . .
+
+# Update the permissions on the testsuite
+RUN chgrp -R 0 /tmp/rhscl_openshift_dir/openjdk && \
+    chmod -R g=u /tmp/rhscl_openshift_dir/openjdk && \
+    chgrp -R 0 /home && \
```
Comment (judovana) [inline ContainerFile:17]: Wou. Why is this file suddenly needed? It is for cpu?

Comment (judovana) [reply ContainerFile:17]: I mena the whole [ContainerFile](https://github.com/rh-openjdk/jdkContainerOcpTests/pull/5/files/e464ad9cabd3fd0d5448d7002a0f19ad3bcb1d65#diff-1f41dac8c7830a4c112e8f6ba376c69c7687fdb7e1c66919a1c12b8749303264) not that tht quoted part

Comment (sefroberg) [reply ContainerFile:17]: The ContainerFile is needed for building the container used for the InterOp testing in the OpenShift CI.
It creates this base image used in their testing: https://quay.io/repository/rhopenjdkqa/rh_jdk_ocp_testsuite

Comment (judovana) [reply ContainerFile:17]: You know better. ty!

**Review by judovana — APPROVED** [review]
IDK

**Review by jmtd — COMMENTED** [review]
LGTM

**Pull request merged**